agent/php_user_instrument.c

```diff
@@ -428,27 +513,40 @@ nruserfn_t* nr_php_add_custom_tracer_callable(zend_function* func TSRMLS_DC) {
   nr_free(name);
 
   nr_php_wrap_zend_function(func, wraprec TSRMLS_CC);
-#if ZEND_MODULE_API_NO < ZEND_7_4_X_API_NO
   nr_php_add_custom_tracer_common(wraprec);
-#endif
 
   return wraprec;
 }
 
+static inline bool nr_php_user_instrument_is_name_valid(const char* namestr,
```

Comment:
This name is a bit misleading - if I pass "$)O#()@::@)@)@@" it will return `true` won't it?  How strongly do we care about the name being "valid"?

Reply:
Or "a::b::c::d" will pass as well.

Reply:
Since there is an API to add these it would be probably good if somewhere these strings are robustly checked - perhaps best in the API code before this level is called?